DEPENDENCIES.md

```diff
@@ -45,7 +45,7 @@ the details of which are reproduced below.
 * [BSD-3-Clause License: sqlcipher](#bsd-3-clause-license-sqlcipher)
 * [Zlib License: adler32](#zlib-license-adler32)
 * [OpenSSL License](#openssl-license)
-* [Optional Notice: SQlite](#optional-notice-sqlite)
+* [Optional Notice: SQLite](#optional-notice-sqlite)
```

Comment:
This is how they capitalize it on their website, so I figure we should follow their lead.